src/DatePicker/CalendarMonth.js

```diff
@@ -38,46 +52,47 @@ class CalendarMonth extends React.Component {
       const selected = !disabled && isSameDate;
 
       if (isSameDate) {
-        if (disabled) {
-          this.selectedDateDisabled = true;
-        } else {
-          this.selectedDateDisabled = false;
-        }
+        this.selectedDateDisabled = disabled;
       }
 
       return (
         <DayButton
-          key={`db${(i + j)}`}
           date={day}
-          onTouchTap={this.handleTouchTap}
-          selected={selected}
           disabled={disabled}
+          key={`db${(i + j)}`}
+          onTouchTap={this.handleTouchTapDay}
+          selected={selected}
         />
       );
     }, this);
   }
 
-  handleTouchTap = (event, date) => {
-    if (this.props.onDayTouchTap) this.props.onDayTouchTap(event, date);
+  styles = {
```

Comment:
@nathanmarks - we do have this in other components when there are no 'dynamic' styles.